lib/psql.js

```diff
@@ -11,11 +11,11 @@ function * exec (db, query) {
   debug(query)
   let env = Object.assign({}, process.env, {
```

Comment:
@dickeyxxx @uhoh-itsmaciek yeah, I believe it is consistent, I would leave as is.